public/app/features/dashboard-scene/sharing/ExportButton/ExportAsJson.tsx

```diff
@@ -0,0 +1,106 @@
+import { css } from '@emotion/css';
+import React from 'react';
+import { useAsync } from 'react-use';
+import AutoSizer from 'react-virtualized-auto-sizer';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { selectors as e2eSelectors } from '@grafana/e2e-selectors';
+import { SceneComponentProps } from '@grafana/scenes';
+import { Button, ClipboardButton, CodeEditor, Label, Spinner, Stack, Switch, useTheme2 } from '@grafana/ui';
+import { Trans } from 'app/core/internationalization';
+
+import { ShareExportTab } from '../ShareExportTab';
+
+const selector = e2eSelectors.pages.ExportDashboardDrawer.ExportAsJson;
+
+export class ExportAsJson extends ShareExportTab {
+  static Component = ExportAsJsonRenderer;
+}
+
+function ExportAsJsonRenderer({ model }: SceneComponentProps<ExportAsJson>) {
+  const theme = useTheme2();
+  const styles = getStyles(theme);
```

Comment:
```suggestion
const styles = useStyles2(getStyles);
```
this already adds the theme inside